internal/proposals/improvement-of-log-messages-usability.md

```diff
@@ -0,0 +1,75 @@
+# Improvement of log messages usability
+
+## Motivation
+
+We want to provide a common, unified solution to logging across Kyma components, which will enable users and maintainers to easily track specific events happening in Kyma's ecosystem. The solution should be independent of the platform (AWS, Azure, GCP) and easy to introduce in existing components
+
+Users, developers, and maintainers want to have an easy way to track events happening in Kyma, problems that occur, and debug faulty components. This will make understanding of Kyma's ecosystem easier and more transparent
+
+## Solution regarding the log messages
+
+Every mission-critical component (or a critical part of the component) must produce log entries in order to make the problem diagnosis process quick and effective
+
+Key assumptions for the log messages:
+
+- Transparency - every entry must be as short as possible but containing all essential information about the process
+- Tracing - entries sequence must be connected so there is an easy way to filter logs concerning a specific operation
+- GDPR-compliance - entries must not contain any sensitive data (password, personal data in the meaning of GDPR)
+- Agile - log messages and their context should be refactored. If the log was useful while debugging the incident, make other also that useful. If there was something missing in the log, report it and refactor
```

Comment:
That would be a nice addition to the onboarding/maintenance guides in Kyma. As it tackles DoD and the way in which we work I would suggest getting rid of that point in this proposal.

Reply:
Deleted. I think it's a good approach though. We should keep this sentence in the guides then